### PR TITLE
RawDatasetLoader turned into abstract class

### PR DESCRIPTION
**Pull request by allaffa**

### CODE REVIEW

**Comment by allaffa** [comment]
@pzhanggit 

I followed your advice: I am assuming that the charge transfer has already been subtracted to the proton number before the data is imported. 
For now, you will see that the class `LSMS_RawDatasetLoader` still has the private method `__charge_density_update_for_LSMS`, but it is not used.

**Review by jychoi-hpc — COMMENTED** [review]
It looks good to me. I have only a minor comment: would you like to clean up unused import statements in "lsms_raw_dataset_loader.py" and "cfg_raw_dataset_loader.py"?

**Comment by allaffa** [comment]
> It looks good to me. I have only a minor comment: would you like to clean up unused import statements in "lsms_raw_dataset_loader.py" and "cfg_raw_dataset_loader.py"?

Yes. That's a good idea. Let me take care of that.

**Review by pzhanggit — COMMENTED** [review]
It looks good to me. After the comment about charge density in lsms loader resolved, I am ready to approve the PR.

**Review by jychoi-hpc — APPROVED** [review]
Thank you for handling this. It looks good to me.

File: tests/deterministic_graph_data.py [hunk 1]
```diff
@@ -130,6 +130,8 @@ def create_configuration(
         knn.fit(positions, node_feature)
         node_output_x = torch.Tensor(knn.predict(positions))
 
+    node_output_x = node_output_x + node_feature
+
     node_output_x_square = node_output_x ** 2
```
Comment (pzhanggit) [inline tests/deterministic_graph_data.py:135]: Could you try something like below?
```suggestion
    node_output_x_square = node_output_x ** 2 + node_feature
```

Comment (allaffa) [reply tests/deterministic_graph_data.py:135]: I already did. But that one also fails.

**Review by pzhanggit — APPROVED** [review]

**Pull request merged**